nautobot_golden_config/filters.py

```diff
@@ -300,9 +300,17 @@ class Meta:
 class ConfigPlanFilterSet(NautobotFilterSet):
     """Inherits Base Class NautobotFilterSet."""
 
-    q = django_filters.CharFilter(
-        method="search",
```

Comment:
The method `search` was undefined? Odd that this didn't raise an exception

Reply:
I think it just got overridden by the redundant q filter down below. That is why it did not raise an exception.

Reply:
Makes sense